features/expo/deviceInfo/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import { Card, Text, View } from 'react-native-ui-lib';
+
+import { useDeviceInfo } from 'htk/features/expo/deviceInfo/hooks';
```

Comment:
Let's use relative imports since this is a repo that is being used as submodule.
This is forcing us to use `htk` path alias. We still did not discuss about the standard path alias.